synthetics.go

```diff
@@ -58,6 +58,12 @@ type SyntheticsOptions struct {
 	MinLocationFailed  *int     `json:"min_location_failed,omitempty"`
 	DeviceIds          []string `json:"device_ids,omitempty"`
 	AcceptSelfSigned   *bool    `json:"accept_self_signed,omitempty"`
+	Retry              *Retry   `json:"retry,omitempty"`
+}
+
+type Retry struct {
+	Count    *int `json:"count,omitempty"`
+	Interval *int `json:"interval,omitempty"`
```

Comment:
They cannot really be omitted in our schema if retry is defined, but consistent with the rest 👍